docs/about.md

```diff
@@ -59,17 +59,44 @@ As the OpenHIM provides these centralised services it means that domain services
-* Acts as a reverse proxy for web service - One can easily configure the OpenHIM to proxy web services, to multiple upstream hosts based on a URL pattern. It also supports multicasting requests to multiple different routes.
-* Secures access to web services - The OpenHIM provides a secure interface to upstream hosts with certificate management and self-signed certificate generation along with advanced access control mechanisms based on client and server certificates.
-* Supports Audit Trail and Node Authentication (ATNA): - The OpenHIM provides a full ATNA audit repository implementation and advanced audit viewer.  
-* Provides visibility into the Service Oriented Architecture (SOA) - The administration console allows viewing of requests as they travel through the system as well as the ability to view metrics such as transaction loads and error rates. It allows an administrator to review and bulk re-run requests or re-run individual requests. The OpenHIM can automatically re-run requests to one's services if client systems are not able to.
-* Extends request processing via mediators - The OpenHIM allows one to build one’s own micro-services, called mediators, that plug into the OpenHIM to extend its functionality. These mediators can be used to transform or orchestrate requests or support more complex business logic. The mediators also report details of what processing has been done back to the OpenHIM using the mediator framework. 
-* Scalable - The OpenHIM is scalable, supporting same server and multi-server clusters, using MongoDB as the underlying database. MongoDB is inherently massively scalable due to its clustering and sharding capabilities and can be horizontally scaled to deal with billions of transactions per day. Sharding increases write performance, while adding secondary nodes to a cluster increases read performance. 
-* Minimal transaction overhead - The OpenHIM used the latest technologies such as Node.js and MongoDB to ensure that it doesn’t introduce any significant overhead to requests.
-
+## Components of the OpenHIM
+
+The OpenHIM logically consists of three components: 
+* The OpenHIM Core provides the main functions and services
+* The Administration Console provides an easy to use interface for system administrators to configure and manage the OpenHIM, giving a window into the workings of the HIE.
+* Mediators are additional services used to extend the functionality of the OpenHIM by transforming and orchestrating transactions.
+
+![OpenHIM Components](/_static/overview/OpenHIM Components.PNG)

```

Comment:
We need to upload the image thats being referenced here. To do so, you can navigate to the `/docs/_static/overview`  directory (while in your banch) and click on the "**upload**" button, Just ensure that there are no spaces in the image name, these will need to be replaced with dashes